lib/xeroizer/exceptions.rb

```diff
@@ -28,6 +28,16 @@ def initialize(type, message, xml, parsed_xml, request_body)
     def message
       "#{@type}: #{@message} \n Generated by the following XML: \n #{@xml}"
     end
+
+    def validation_errors
+      errors = []
+      @parsed_xml.xpath("//ValidationError").each do |err|
+        errors = errors | [err.text.gsub(/^\s+/, '').gsub(/\s+$/, '')]
```

Comment:
I took that approach to prevent duplicate error messages that I encountered whilst testing